test_agi_check.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+def agi_check(stat_c, stat_p):
```

Comment:
You shouldn't need/want to define the function again in your test file. So you'll need to import it:
```
from arena import agi_check
```